### PR TITLE
Updates to RoI transforms docs

### PR DESCRIPTION
**Pull request by NicolasHug**
This PR adds some minor edits to the docstrings of the RoI transforms, in particular about the feature map indices in the RoIs tensor which were previously a source of confusion to me

### CODE REVIEW

File: torchvision/ops/ps_roi_align.py [hunk 1]
```diff
@@ -19,23 +19,26 @@ def ps_roi_align(
     mentioned in Light-Head R-CNN.
 
     Args:
-        input (Tensor[N, C, H, W]): input tensor
+        input (Tensor[N, C, H, W]): The input tensor, i.e. a batch with ``N`` feature maps
```
Comment (fmassa) [inline torchvision/ops/ps_roi_align.py:22]: This is actually not correct -- N is the batch size, C is the number of feature maps (aka channels)

File: torchvision/ops/ps_roi_align.py [hunk 2]
```diff
@@ -19,23 +19,26 @@ def ps_roi_align(
...
             format where the regions will be taken from.
             The coordinate must satisfy ``0 <= x1 < x2`` and ``0 <= y1 < y2``.
-            If a single Tensor is passed,
-            then the first column should contain the batch index. If a list of Tensors
-            is passed, then each Tensor will correspond to the boxes for an element i
-            in a batch
-        output_size (int or Tuple[int, int]): the size of the output after the cropping
-            is performed, as (height, width)
+            If a single Tensor is passed, then the first column should
+            contain the index of the corresponding feature map in the batch, i.e. a number in ``[0, N - 1]``.
```
Comment (fmassa) [inline torchvision/ops/ps_roi_align.py:27]: index of the corresponding element in the batch. I think calling it feature map might be more more confusing

**Review by fmassa — CHANGES_REQUESTED** [review]
Thanks for the PR!

I think some parts might be a bit misleading now, could you have a look?

File: torchvision/ops/ps_roi_align.py (hunk 2 above)
Comment (NicolasHug) [reply torchvision/ops/ps_roi_align.py:27]: ah thanks, I was assuming a feature map was `C x H x W` for some reason

**Review by fmassa — APPROVED** [review]
Thanks!

**Pull request merged**